src/irc/ConnectionInstance.ts

```diff
@@ -330,7 +330,9 @@ export class ConnectionInstance {
 
     private listenForCTCPVersions() {
         this.client.addListener("ctcp-version", (from: string) => {
```

Comment:
If string can be falsey, this signature is incorrect.

Reply:
Not changing the signature?
This `from: string` should likely be `from: string|undefined` or ``from: string|null`.